### PR TITLE
fix: alert on duplicate name virtual view creation

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: https://github.com/lightdash/lightdash/issues/11820

### Description:

Screenshot 2024-10-07 at 13 17 35

Screenshot 2024-10-07 at 13 27 26

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx [hunk 1]
```diff
@@ -117,6 +120,7 @@ export const CreateVirtualViewModal: FC<Props> = ({ opened, onClose }) => {
                         required
                         // TODO - don't allow duplicate names
                         {...form.getInputProps('name')}
+                        error={!!error?.error}
```
Comment (joaoviana) [inline packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx:123]: I tried to do a `instaceof AlreadyExistsError` but it was always returning `false`, so I've only enabled the red border (we display the error message on the toast instead)

File: packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx [hunk 2]
```diff
@@ -117,6 +120,7 @@ export const CreateVirtualViewModal: FC<Props> = ({ opened, onClose }) => {
                         required
                         // TODO - don't allow duplicate names
```
Comment (rephus) [inline packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx:121]: remove TODO ?

Comment (joaoviana) [reply packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx:121]: 🙌

Comment (joaoviana) [reply packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx]: removed: https://github.com/lightdash/lightdash/pull/11833/commits/95f83aa67efd06f5108f5f480b08c62f077a32fc

**Review by rephus — APPROVED** [review]

**Pull request merged**